modules/mobile/spec/request/claims_document_upload_spec.rb

```diff
@@ -86,7 +88,7 @@
   end
 
   it 'normalizes requests with a null tracked_item_id' do
-    params = { file:, claim_id:, tracked_item_ids: 'null', documentType: document_type }
+    params = { file:, claim_id:, tracked_item_id: nil, documentType: document_type }
```

Comment:
we had a test all along for tracked item id but it was only testing for nil which it always was but we never tested for a present one. the string 'null' doesn't seem like a valid case as I'm unsure if the id is an integer or not so am hesitant to try to force it into one. passing nil makes the parameter an empty string (which seems weird) but I added some safety guards against it so we don't pass it to the LH service